.github/workflows/test.yaml

```diff
@@ -0,0 +1,20 @@
+on: [push, pull_request]
+name: Build and Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x]
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
```

Comment:
It might be a good idea to add a [actions/cache@v2](https://github.com/actions/cache) for `go env GOCACHE`? Or wherever `go get` downloads things.